integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bindings/python:build_defs.oss.bzl",
+    "INTREE_TENSORFLOW_PY_DEPS",
+    "NUMPY_DEPS",
+    "iree_py_test",
+    "iree_py_test_suite",
+    "iree_vision_test_suite",
+)
+
+E2E_INTEGRATIONS_DEPS = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+    "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+]
+
+iree_py_test_suite(
+    name = "non_vision",
+    srcs = glob(["*_test.py"]),
+    excludes = [
+        "keras_vision_model_test.py",
+        # TODO(GH-1620): Re-enable after fixing the failure on GitHub Actions.
+        "keras_lstm_static_test.py"
+    ],
+    python_version = "PY3",
+    deps = E2E_INTEGRATIONS_DEPS,
+)
+
+iree_vision_test_suite(
+    name = "keras_vision_models",
+    configurations = [
+        # tuples of (data, include_top, model_name, backends)
+        # "cifar10" has toy models with input 32x32, is good for debugging
+        ("cifar10", 1, "ResNet50", "tf,iree_vmla"),
+        ("cifar10", 1, "ResNet50", "tf,iree_llvmjit"),
+        ("cifar10", 1, "ResNet50", "tf,iree_vulkan"),
+    ],
+    deps = E2E_INTEGRATIONS_DEPS,
+    tags = ["manual"],
+)
+
+iree_vision_test_suite(
+    name = "keras_vision_models_external",
+    configurations = [
+        # tuples of (data, include_top, model_name, backends)
+        # "cifar10" has toy models with input 32x32, is good for debugging
+        # "imagenet" has real model weights for input 224x224
+        # TODO(rybakov) remove tests with include_top=0 as soon as include_top=1 works
+        ("cifar10", 1, "ResNet50", "tf"),
+        ("cifar10", 0, "MobileNet", "tf,iree_vmla"),
+        ("cifar10", 1, "MobileNet", "tf,iree_vmla"),
+        ("cifar10", 0, "MobileNetV2", "tf"),
+        ("cifar10", 1, "MobileNetV2", "tf"),
+        ("imagenet", 0, "ResNet50", "tf"),
+        ("imagenet", 1, "ResNet50", "tf"),
+        ("imagenet", 0, "MobileNet", "tf"),
+        ("imagenet", 1, "MobileNet", "tf"),
+        ("imagenet", 0, "MobileNetV2", "tf"),
+        ("imagenet", 1, "MobileNetV2", "tf"),
+    ],
+    external_data = True,
+    deps = E2E_INTEGRATIONS_DEPS,
+    tags = [
+        "external",
+        "large",
```

Comment:
While you're here, this tag does nothing
```suggestion
```

Reply:
Good to know. I just copied it over from the current head.

Reply:
Yeah I know. It's been bothering me :-D